app/app.go

```diff
@@ -111,12 +120,30 @@ import (
 	microtxtypes "github.com/althea-net/althea-chain/x/microtx/types"
 )
 
-const appName = "app"
+const appName = "althea"
 
-var (
-	// DefaultNodeHome default home directories for the application daemon
-	DefaultNodeHome string
+// DefaultNodeHome default home directories for the application daemon
+var DefaultNodeHome string
+
+func init() {
+	// Set DefaultNodeHome before the chain starts
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
+	}
+	DefaultNodeHome = filepath.Join(userHomeDir, ".althea")
+
```

Comment:
What do you think a sensible value here is?

Reply:
.althea is fine